lib/index.js

```diff
@@ -90,6 +118,13 @@ function createHttpClient(options) {
 }
 
 
+/**
+ * creates a server.
+ * @public
+ * @function createServer
+ * @param    {Object} options an options object
```

Comment:
Yeah, I experimented with generating the JSDocs, but they're super wonky, and there are lots of corner cases which are less than ideal. My instinct is to keep the JSDoc comments in place purely for developer use (which is always good), and continue to shore up user docs as a separate thing.